src/app/map/[id]/components/Map.tsx

```diff
@@ -637,6 +651,23 @@ export default function Map({
       <div className="absolute top-4 right-4 z-20">
         <SearchBox />
       </div>
+      {view && view.isTag && !isTableOpen && (
+        <div
+          className="absolute top-4 z-20 transition-transform duration-300 w-full"
+          style={{
+            left: showControls ? `${CONTROL_PANEL_WIDTH + 16}px` : "40px",
+          }}
+        >
+          <TagExplainerCard
+            onConfigureTag={onConfigureTag || (() => undefined)}
+            onResendTags={handleResendTags}
+            dataSource={firstDataSource}
+            dataSourceView={firstDataSourceView}
+            viewName={view?.name}
+            placedMarkers={placedMarkers}
+          />
+        </div>
+      )}
```

Comment:
Could we move it to MapWrapper? That's where the elements with position depending on the sidebar live and there's a little helper

`  const positionLeft = {
    transform: showControls
      ? `translateX(calc(${CONTROL_PANEL_WIDTH}px))`
      : "translateX(0)",
  };`
  
  The Map component is already giantic so if we can avoid adding to it, that's what I would suggest :)